index.html

```diff
@@ -0,0 +1,401 @@
+                                                </p>
+                                        <a href="#blank">                            Should You Spend, or Save, as if You’ll Live Forever?
+                                            </a>
+                                        <p>Sept. 12</p>
+                                </div>
+                                <div class="box j">
+                                        <img src="https://static01.nyt.com/images/2019/09/12/multimedia/12sp-Ret-Cover-illo/12sp-Ret-Cover-illo-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                                        <p>Margeaux Walter for The New York Times</p>
+                                        <a href="#blank">An Open Letter (and a Rant) on Age Discrimination at Work</a>
+                                        <p>Sept. 12</p>
+                                </div>
+                          </div>
+            </div>
+        
+                
+            <aside class="aside">
+                <hr>
+                <h2>Most Popular</h2>
+                <h2>36 Hours in Albuquerque</h2>
+                <hr>
+                <h2>Three Junior-Fours on the East Side. Which Did They Choose?</h2>
+                <hr>
+                <h2>Young People Are Going to Save Us All From Office Life</h2>
+                <hr>
+                <h2>Justin Trudeau Says He Is ‘Deeply Sorry’ After Brownface Photo Surfaces</h2>
+                <hr>
+                <h2>Whistle-Blower’s Complaint Is Said to Involve Multiple Acts by Trump</h2>
+                <hr>
+                <h2>Opinion: Barack Obama’s Biggest Mistake</h2>
+                <hr>
+                <h2>Can the N.F.L. Turn a 360-Pound Rugby Player Into a Football Star?</h2>
+                <hr>
+                <h2>What Really Brought Down the Boeing 737 Max?</h2>
+                <hr>
+                <h2>‘Ad Astra’ Review: Brad Pitt Orbits the Powers of Darkness</h2>
+                <hr>
+                <h2>New Video Surfaces Showing Trudeau in Blackface, Compounding Scandal</h2>
+                <hr>
+    
+            </aside>
+        </section>
+       <nav class="site-index">
+    
+        <div>
+            <div class="float-left">
+                <img class="logo-name float-left" src="./public/The_New_York_Times_logo.png" alt="tnyt">
+            </div>
+            <a class="float-right" href="#" >Go to Home Page »</a>
+            <hr class="clear-both">
+        </div>
+        <div class="clear-both">
+            <h3>NEWS</h3>
+            <hr>
+        </div>
+        <div>
+            <h3>OPNION</h3>
+            <hr>
+        </div>
+        <div>
+            <h3>ARTS</h3>
+            <hr>
+        </div>
+        <div>
+            <h3>LIVING</h3>
+            <hr>
+        </div>
+        <div>
+            <h3>LISTINGS & MORE</h3>
+            <hr>
+        </div>
+    
+       </nav>
+    
+       <footer >
```

Comment:
Make this footer look close the original as possible.

Reply:
hey man thanks for the feedback we are fixing right now, but we dont understand what wh need to chenge from the foooter, could you explain it more to us ?, thanks